_docs/tasks/security/mutual-tls.md

```diff
@@ -59,25 +59,25 @@ Istio CA is up if the "AVAILABLE" column is 1.
 When running Istio with mutual TLS authentication turned on, you can use curl in one service's
 envoy to send request to other services.
 For example, after starting the [BookInfo]({{home}}/docs/guides/bookinfo.html) 
-sample application you can ssh into the envoy container of `productpage` service, 
+sample application you can ssh into the envoy container of `ratings` service,
```

Comment:
curious why the change ? should work the same no ?

Reply:
See my previous comment in this thread: "Currently somehow only productpage works, so this PR is for temporary workaround. I am looking into the problem."

Reply:
It's reverted, and turns out the endpoint changes.